src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/tests/__snapshots__/SortableBlock.test.js.snap

```diff
@@ -66,10 +66,12 @@ exports[`Render expanded sortable, non-collapsable block with types 1`] = `
         class="types"
       >
         <div
+          aria-label="select"
```

Comment:
We should not add here a aria-label. At current state we should keep the components as they are. And use `classes` and `>` selector to check for specific component. to exist.

```suggestion
```

Reply:
@alexander-schranz I disagree here a bit ... It definitely doesn't make sense to add aria-label to lots of tags (including this specific example here), but if there are buttons without a button text with just an icon, we should add a useful `aria-label`, which describes, what the button does. That label should of course also be translated. Adding a useful aria-label definitely doesn't hurt and makes writing some tests a lot easier.

Reply:
But yes, if the button already has a button text, it's definitely not necessary, because jest allows to select buttons by their text easily

Reply:
@b3h3m0th if you run into any way where you can currently not select or a aria-label would make selection easier for the tests. Add a comment there so we can have a look at together and discuss how to forward with that specific tests. Aria-Label would require always come from the backend translated, also im some other cases Aria-role or tag selection (button, ...) could be used. Most of the selection but should be done via Text if possible as that would make it most future proof also when we are refractoring the html and accessibility.

Reply:
Sure, make sense. Thanks!